doc/source/users_guide/multi-instance.rst

```diff
@@ -47,16 +45,21 @@ To run two instances of CAM, CLM, CICE, RTM and DOCN, invoke the following comma
    > ./xmlchange NINST_ROF=2
    > ./xmlchange NINST_OCN=2
 
-As a result, you will have two instances of CAM, CLM and CICE (prescribed), RTM, and DOCN, each running concurrently on 64 MPI tasks.
+As a result, you will have two instances of CAM, CLM and CICE (prescribed), RTM, and DOCN, each running concurrently on 72 MPI tasks and all using the same driver/coupler component.   In this single driver/coupler mode the number of tasks for each component instance is NTASKS_COMPONENT/NINST_COMPONENT and the total number of tasks is the same as for the single instance case.
 
-**TODO: put in reference to xmlchange".**
+Now consider the multi driver model.
+To use this mode change
+::
+   > ./xmlchange MULTI_DRIVER=TRUE
+
+This configuration will run each component instance on the original 144 tasks but will generate two copies of the model (in the same executable) for a total of 288 tasks.
 
 3. Set up the case
 ::
 
    > ./case.setup
 
-A new **user_nl_xxx_NNNN** file (where NNNN is the number of the component instances) is generated when **case.setup** is called. 
+A new **user_nl_xxx_NNNN** file (where NNNN is the number of the component instances) is generated when **case.setup** is called.
 When calling **case.setup** with the **env_mach_pes.xml** file specifically, these files are created in **$CASEROOT**:
 ::
```

Comment:
I don't understand why you are commenting on code and comments that I haven't changed.   I think you are correct but are they really issues for this PR or should they be noted separately?